.github/workflows/e2e_cron.yml

```diff
@@ -49,7 +48,7 @@ jobs:
       - name: Run Playwright tests - ${{ inputs.environment != null && inputs.environment || 'stage' }}
         if: github.actor != 'dependabot[bot]'
         run: npm run e2e
-        timeout-minutes: 20
+        timeout-minutes: 30
```

Comment:
Do we need a suite timeout, or could we remove it? (Random q: not PR blocking)

Reply:
Seems like we do because tests are stuck idling when they reach certain states. See https://github.com/mozilla/blurts-server/actions/runs/9421692534, the application error state on the webpage causes the tests to get stuck? Not sure why the job keeps running.

Also not sure why thats happening because we have a test timeout limit here, https://github.com/mozilla/blurts-server/blob/main/playwright.config.js#L27-L28.